code/game/objects/items/melee/misc.dm

```diff
@@ -133,6 +133,28 @@
 	if(!originally_nodropped)
 		item_flags &= ~NODROP
 
+/obj/item/melee/rapier
+	name = "plastitanium rapier"
+	desc = "A impossibly thin blade made of plastitanium with a tip made of diamond. It looks to be able to cut through any armor."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "rapier"
+	item_state = "rapier"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	force = 33
+	throwforce = 45
```

Comment:
this + 100 AP means you just pair it with TK in crit people in 3 clicks.

Reply:
_33*3 = 99 tho,
also, on Cita, you aren't in crit until you get 120 damage _

Reply:
Hm... it should probably have a good chance to embed.

Reply:
No